### PR TITLE
MYB-65 RealmHelper created to save and get the favorite locations

### PR DESCRIPTION
**Pull request by jcg22cabeza**

### CODE REVIEW

File: app/src/main/java/com/mybus/dao/RealmDao.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.mybus.dao;
+
+import com.mybus.model.FavoriteLocation;
```
Comment (ldimitroff) [inline app/src/main/java/com/mybus/dao/RealmDao.java:3]: Unused Import

File: app/src/main/java/com/mybus/model/FavoriteLocation.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import io.realm.annotations.Required;
+
+/**
+ * Created by Julian Gonzalez <jgonzalez@devspark.com>
+ * <p/>
+ * RealmObject to persist favorite locations
+ */
+public class FavoriteLocation extends RealmObject {
+    @PrimaryKey
+    private String name;
```
Comment (ldimitroff) [inline app/src/main/java/com/mybus/model/FavoriteLocation.java:14]: PrimaryKey should be a number and unique identifyer

Comment (quiquep) [inline app/src/main/java/com/mybus/model/FavoriteLocation.java:14]: It looks like with realm the only way is declaring the name as primary key (the unique constraint is not supported yet). In that case the name should be a read-only property for the object (the setter should be removed).

**Comment by jcg22cabeza** [comment]
I will open a new PR when refactor is finished.

**Pull request closed without merging**